data/announcements/scheduled.yaml

```
@@ -30,6 +30,15 @@
 # leave the "announcements" key in place
 announcements:
 
+- name: Dockershim removal
+  startTime: 2022-04-13T00:00:00 # Added in PR: https://github.com/kubernetes/website/pull/32805
+  endTime: 2022-05-19T00:00:00 # Expires 2022-05-19
```

Thoughts on keeping the banner up until v1.25 is released?

```suggestion
  endTime: 2022-08-09T00:00:00 # Expires 2022-08-09
```

---

There was a suggestion from @dims to replace this banner right after 1.24 goes out with a: "kubecon EU / contributor stuff banner". Any other thoughts?

---

I think it's safe to leave it up for a long time. But, when there are better things to put in there, we should. My two cents.

---

I'd lean towards leaving it up there personally, a contributor banner might be more appropriate for k8s.dev

---

A few weeks around the release of 1.24 seems better to me. It's a *really* intrusive banner that I don't think is relevant to the majority of people visiting kubernetes.io for 4 months. A prominent section at the top of kubelet / CRI config-oriented pages would be more targeted.

---

Okay. Let's pick some dates. How about we start the Dockershim banner as soon as this PR merges and end it on April 30, 2022. Then we could run the Kubecon EU banner from May 1 until it's over.

---

Also, we could do both. Have a banner everywhere for a week or two before and after the release, then move it to just the kubelet/CRI pages for a while after that. Thoughts?

---

@chrisnegus if you want a _banner_ on specific pages, you'd have to implement that. We can add something else, more durably, to specific pages (eg the shortcode from #32826)

---

I'm suggest that we have this "really intrusive" (I agree) banner on the docs site only be there from now until one week after the release comes out. Then we can use the shortcodes on specific pages for a longer period of time (see [#32826](https://github.com/kubernetes/website/pull/32826)). I'm looking for agreement on the banner wording and timeframe for the time being.